.github/labeler.yml

```
@@ -229,6 +229,14 @@ team-infra:
     - any-glob-to-any-file:
       - docs/infra/**/*
 
+team-ios:
+  - changed-files:
+    - any-glob-to-any-file:
+      - engine/src/flutter/shell/platform/darwin/common/**/*
+      - engine/src/flutter/shell/platform/darwin/ios/framework/**/*
+      - packages/flutter_tools/**/ios/*
+      - packages/flutter_tools/**/*xcode*
```

Perhaps also anything with `ios` or `macos` in the name? So we get files like [build_system/targets/ios.dart](https://github.com/flutter/flutter/blob/master/packages/flutter_tools/lib/src/build_system/targets/ios.dart) and [build_system/targets/macos.dart](https://github.com/flutter/flutter/blob/master/packages/flutter_tools/lib/src/build_system/targets/macos.dart).

```suggestion
      - packages/flutter_tools/**/*xcode*
      - packages/flutter_tools/**/*ios*
      - packages/flutter_tools/**/*macos*
```